vllm/worker/hpu_model_runner.py

```diff
@@ -1978,7 +1978,9 @@ def warmup_scenario(self,
         if profiler:
             profiler.stop()
         self.profiler.end()
-        gc.collect()
+        # don't do gc for dummy run
+        if not (batch_size == 1 and seq_len == 128):
```

Comment:
I think it's more clear to add an extra parameter like `is_dummy_run`

Reply:
nice catch , updated.